suite/tests/CMakeLists.txt

```diff
@@ -2874,6 +2874,7 @@ endif ()
       torunonly_drcachesim(filter-simple ${ci_shared_app} "-L0_filter" "")
       torunonly_drcachesim(filter-no-i ${ci_shared_app} "-L0_filter -L0I_size 0" "")
       torunonly_drcachesim(filter-no-d ${ci_shared_app} "-L0_filter -L0D_size 0" "")
+      torunonly_drcachesim(filter-inst-only ${ci_shared_app} "-inst_only_trace" "")
```

Comment:
Please also add an offline-trace test to make sure post-processing of raw traces works when there are no data references (raw2trace may have some issues).

Reply:
It seems that when -offline is present, the trace somehow continues to have data references. Checking.

Reply:
Offline trace collection is fixed now. Also, added the suggested test.